<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::WebsiteInformationDialog</class>
 <widget class="QDialog" name="Otter::WebsiteInformationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="hostLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="stateLayout">
     <item>
      <widget class="QLabel" name="stateIconLabel">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="stateLabel"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="generalLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="informationHeaderLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Information</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="addressLabel">
         <property name="text">
          <string>Address:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="encodingLabel">
         <property name="text">
          <string>Encoding:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="sizeLabel">
         <property name="text">
          <string>Size:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="elementsLabel">
         <property name="text">
          <string>Elements:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="downloadDateLabel">
         <property name="text">
          <string>Download date:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Otter::TextLabelWidget" name="addressLabelWidget" native="true"/>
       </item>
       <item row="3" column="1">
        <widget class="Otter::TextLabelWidget" name="encodingLabelWidget" native="true"/>
       </item>
       <item row="4" column="1">
        <widget class="Otter::TextLabelWidget" name="sizeLabelWidget" native="true"/>
       </item>
       <item row="5" column="1">
        <widget class="Otter::TextLabelWidget" name="elementsLabelWidget" native="true"/>
       </item>
       <item row="6" column="1">
        <widget class="Otter::TextLabelWidget" name="downloadDateLabelWidget" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>Title:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="Otter::TextLabelWidget" name="titleLabelWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="permissionsTab">
      <attribute name="title">
       <string>Permissions</string>
      </attribute>
      <layout class="QFormLayout" name="permissionsLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="preferencesHeaderLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Preferences</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="preferencesLayout">
         <item>
          <spacer name="preferencesSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="preferencesDetailsButton">
           <property name="text">
            <string>Details…</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="cookiesLabel">
         <property name="text">
          <string>Set cookies:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="thirdPartyCookiesLabel">
         <property name="text">
          <string>Set third-party cookies:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="notificationsLabel">
         <property name="text">
          <string>Show notifications:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="geolocationLabel">
         <property name="text">
          <string>Access your location:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="pluginsLabel">
         <property name="text">
          <string>Load plugins:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="imagesLabel">
         <property name="text">
          <string>Load images:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="javascriptLabel">
         <property name="text">
          <string>Use JavaScript:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="cookiesValueLabel"/>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="thirdPartyCookiesValueLabel"/>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="pluginsValueLabel"/>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="imagesValueLabel"/>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="javascriptValueLabel"/>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="geolocationValueLabel"/>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="notificationsValueLabel"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="popupsLabel">
         <property name="text">
          <string>Show pop-up windows:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="popupsValueLabel"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="fullScreenLabel">
         <property name="text">
          <string>Enter full screen mode:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="fullScreenValueLabel"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="securityTab">
      <attribute name="title">
       <string>Security</string>
      </attribute>
      <layout class="QVBoxLayout" name="securityLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="certificateHeaderLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Certificate</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="certificateLayout">
           <item>
            <spacer name="certificateSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="certificateDetailsButton">
             <property name="text">
              <string>Details…</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="certificateIissuedToLabel">
           <property name="text">
            <string>Issued to:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="certificateIssuedByLabel">
           <property name="text">
            <string>Issued by:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="certificateIssuedOnLabel">
           <property name="text">
            <string>Issued on:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="certificateExpiresOnLabel">
           <property name="text">
            <string>Expires on:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="cipherHeaderLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Cipher</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="cipherProtocolLabel">
           <property name="text">
            <string>Protocol:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="cipherAuthenticationMethodLabel">
           <property name="text">
            <string>Authentication method:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="cipherEncryptionMethodLabel">
           <property name="text">
            <string>Encryption method:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="cipherKeyExchangeMethodLabel">
           <property name="text">
            <string>Key exchange method:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="Otter::TextLabelWidget" name="certificateIssuedByLabelWidget" native="true"/>
         </item>
         <item row="1" column="1">
          <widget class="Otter::TextLabelWidget" name="certificateIssuedToLabelWidget" native="true"/>
         </item>
         <item row="3" column="1">
          <widget class="Otter::TextLabelWidget" name="certificateIssuedOnLabelWidget" native="true"/>
         </item>
         <item row="4" column="1">
          <widget class="Otter::TextLabelWidget" name="certificateExpiresOnLabelWidget" native="true"/>
         </item>
         <item row="6" column="1">
          <widget class="Otter::TextLabelWidget" name="cipherProtocolLabelWidget" native="true"/>
         </item>
         <item row="7" column="1">
          <widget class="Otter::TextLabelWidget" name="cipherAuthenticationMethodLabelWidget" native="true"/>
         </item>
         <item row="8" column="1">
          <widget class="Otter::TextLabelWidget" name="cipherEncryptionMethodLabelWidget" native="true"/>
         </item>
         <item row="9" column="1">
          <widget class="Otter::TextLabelWidget" name="cipherKeyExchangeMethodLabelWidget" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="sslErrorsHeaderLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>SSL Errors</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Otter::ItemViewWidget" name="sslErrorsViewWidget">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>200</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::ItemViewWidget</class>
   <extends>QTreeView</extends>
   <header>src/ui/ItemViewWidget.h</header>
  </customwidget>
  <customwidget>
   <class>Otter::TextLabelWidget</class>
   <extends>QWidget</extends>
   <header>src/ui/TextLabelWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Otter::WebsiteInformationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
